h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamMojoModelBase.java

```diff
@@ -51,13 +54,25 @@
   double[][] _hj;   // difference between knot values
   int _numExpandedGamCols; // number of expanded gam columns
   int _lastClass;
+  boolean _inputContainGamCols; // true if input dataset contains gam column names
   
   private int getTotFeatureSize() { return _totFeatureSize;}
   
   GamMojoModelBase(String[] columns, String[][] domains, String responseColumn) {
     super(columns, domains, responseColumn);
   }
 
+  void extractInputDataNames(String[] columnNames) {
```

Comment:
could use a better name, since the current name has nothing to do what this does, maybe `checkNamesContainsGamCols`?

Reply:
I removed the function.